zio-http/src/main/scala/zio/http/Route.scala

```diff
@@ -0,0 +1,486 @@
+package zio.http
+
+import io.netty.handler.codec.http.HttpHeaderNames
+import zio._
+import zio.http.model.{Headers, MediaType, Status}
+import zio.http.socket.{SocketApp, WebSocketChannelEvent}
+import zio.stream.ZStream
+
+import java.io.{File, FileNotFoundException}
+import java.nio.file.Paths
+import java.util.zip.ZipFile
+
+trait Route[-R, +Err, -In, +Out] { self =>
+  protected def run(in: In): HExit[R, Err, Handler[R, Err, In, Out]]
+
+  final def >>>[R1 <: R, Err1 >: Err, In1 >: Out, Out1](
+    handler: Handler[R1, Err1, In1, Out1],
+  ): Route[R1, Err1, In, Out1] =
+    Route.fromHandlerHExit[In] { in =>
+      self.run(in).map { firstHandler =>
+        if (firstHandler eq null) null.asInstanceOf[Handler[R1, Err1, In, Out1]]
+        else firstHandler.andThen(handler)
+      }
+    }
+
+  final def @@[R1 <: R, Err1 >: Err, In1 <: In, Out1 >: Out, In2, Out2](
+    aspect: RouteAspect[R1, Err1, In1, Out1, In2, Out2],
+  ): Route[R1, Err1, In2, Out2] =
+    aspect(self)
+
+  final def @@[R1 <: R, Err1 >: Err, In1 <: In, Out1 >: Out, In2 <: In, Out2](
+    aspect: HandlerAspect[R1, Err1, In1, Out1, In2, Out2],
+  ): Route[R1, Err1, In2, Out2] =
+    Route.fromHandlerHExit[In2] { in =>
+      self.run(in).flatMap { handler =>
+        if (handler eq null) HExit.succeed(null)
+        else HExit.succeed(aspect(handler))
+      }
+    }
+
+  final def ++[R1 <: R, Err1 >: Err, In1 <: In, Out1 >: Out](
+    that: Route[R1, Err1, In1, Out1],
+  ): Route[R1, Err1, In1, Out1] =
+    self.defaultWith(that)
+
+  final def defaultWith[R1 <: R, Err1 >: Err, In1 <: In, Out1 >: Out](
```

Comment:
This might be called `orElse` as it represents a kind of fallback.

Reply:
I agree, I was following the old API here